backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -482,6 +486,42 @@ export class AdminResolver {
       maxPerCycle,
     }: ContributionLinkArgs,
   ): Promise<ContributionLink> {
+    logger.debug(
+      `createContributionLink(validFrom=${validFrom}, validTo=${validTo}, name=${name}, amount=${amount}, memo=${memo}, cycle=${cycle}, maxPerCycle=${maxPerCycle}, maxAmountPerMonth=${maxAmountPerMonth})...`,
+    )
+    if (!isStartEndDateValid(validFrom, validTo)) {
+      logger.error(`The validFrom=${validFrom} must be before or equals the validTo=${validTo}!`)
+      throw new Error(`The validFrom=${validFrom} must be before or equals the validTo=${validTo}!`)
+    }
+    if (name == null) {
```

Comment:
```suggestion
    if (!name) {
```
Should never happen, as graphql validation already failed.

Reply:
adopt the suggestion